projects/json-c/tokener_parse_ex_fuzzer.cc

```diff
@@ -0,0 +1,14 @@
+#include <stdint.h>
+
+#include <json.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const char *data, size_t size) {
+  json_tokener *tok = json_tokener_new();
+  json_object *obj;
```

Comment:
why not call `json_tokener_parse_ex` from line 9 right here?

Reply:
No reason at all. Will change.